yarn.lock

```diff
@@ -17124,18 +17140,6 @@ react-dnd-html5-backend@11.1.3:
   dependencies:
     dnd-core "^11.1.3"
 
-react-dnd-test-backend@^11.1.3:
```

Comment:
Thanks for this - I think this wasn't removed in my PR #1941 because of an issue with node-gyp on my machine.